app/models/mixins/ui_service_mixin.rb

```diff
@@ -1,21 +1,24 @@
 module UiServiceMixin
+  # rubocop:disable Metrics/LineLength
```

Comment:
I think the line length is one of the rules, that is more like "advisory" in nature - try to stay shorter, but if it does not work then it is not bad either.
I do not know though if it is better practice to just accept the violation or add a comment like you did @Jiri-Kremser - @Fryguy wdyt?